modules/gitops-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-6_{context}"]
+= Release notes for {gitops-title} 1.8.6
+
+{gitops-title} 1.8.6 is now available on {OCP} 4.10, 4.11, 4.12, and 4.13.
+
+[id="errata-updates-1-8-6_{context}"]
+== Errata updates
+
+[id="gitops-1-8-6-security-update-advisory_{context}"]
+=== RHSA-2023:XXXX - {gitops-title} 1.8.6 security update advisory
+
+Issued: 2023-11-07
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://errata.engineering.redhat.com/advisory/122175[RHSA-2023:XXXX]
```

Comment:
This link format will be:
```suggestion
* link:https://access.redhat.com/errata/RHSA-2023:XXXX[RHSA-2023:XXXX]
```
Suggest using that instead of the internal-only link you have here. That way, when you have the Live ID, you can just replace `XXXX` throughout :slightly_smiling_face:

Reply:
Alright, noted. thank you for the heads up.